File: In_Javascript/ex12.js
// Rejim
function rejim(x, y, z) {
    if (x < y && y < z) {
        return "Qattiq rejim";
    } else if (x <= y && y <= z) {
        return "Yumshoq rejim";
    } else {
        return "Bu hato";
    }
}

console.log(rejim(1, 2, 3));

File: In_Ruby/ex12.rb
# rejim
def rejim(x, y, z)
  if x < y && y < z
    return "Qattiq rejim"
  elsif x <= y && y <= z
    return "Yumshoq rejim"
  else
    return "Bu hato"
  end
end

puts rejim(10, 15, 31)